class-plugin.php

```diff
@@ -33,6 +39,28 @@ public function action_admin_init() {
 
 		register_setting( 'general', 'hm_gtm_id', 'sanitize_text_field' );
 
+		// multisite field below
+		if ( ! is_multisite() ) {
+			return;
+		}
+
+		// add settings for primary blog
+		if ( SITE_ID_CURRENT_SITE === get_current_blog_id() ) {
```

Comment:
I suppose to display the setting only on the main site. In order bloggers couldn't modify the field. Is it wrong?